docs/docs/reference/soft-modifier.md

```diff
@@ -9,7 +9,6 @@ TODO this is most likely outdated should at least contain `extension` in additio
 Worth maintaining? or maybe better refer to internal/syntax.md ? 
 -->
 
-It is treated as a potential modifier of a definition, if it is followed by a hard modifier or a keyword combination starting a definition (`def`, `val`, `var`, `type`, `class`, `case class`, `trait`, `object`, `case object`, `enum`). Between the two words there may be a sequence of newline tokens and soft modifiers.
+It is treated as a potential modifier of a definition if it is followed by a hard modifier or a keyword combination starting a definition (`def`, `val`, `var`, `type`, `class`, `trait`, `object`, `enum`, `given`). Between the two words there may be a sequence of newline tokens and soft modifiers. In addition, `inline` is a keyword for `inline if`, `inline ... match`, and inline parameters.
```

Comment:
Are `case object` and `case class` removed by accident?

Reply:
I removed them on purpose. `case class` definitions are `class` definitions. Having both seemed to be redundant. The same applies to `object` and `case object`.

Reply:
The file `Tokens.scala` contains the definitions `defIntroTokens` and `templateIntroTokens`. It might be a little better to align with the definitions there.